.changeset/dull-bees-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rsbuild/plugin-css-minimizer': patch
+---
+
+export CssMinimizerWebpackPlugin for user configuration
```

Comment:
Rsbuild repo do not requires a changeset, we uses GitHub releases page to generate changelog (by commit info)

Reply:
So does the changeset need to be deleted?

Reply:
No, it is still used to bump the package version. We plan to remove it in the long term.